src/Plugins/RpcServer/RpcError.cs

```diff
@@ -62,6 +62,10 @@ public class RpcError
         public static readonly RpcError ExpiredTransaction = new(-510, "Expired transaction");
         public static readonly RpcError InsufficientFunds = new(-511, "Insufficient funds for fee");
         public static readonly RpcError InvalidContractVerification = new(-512, "Invalid contract verification function");
+        public static readonly RpcError InvalidTransaction = new(-513, "Invalid transaction");
+        public static readonly RpcError UnableToVerify = new(-514, "Unable to verify transaction");
+        public static readonly RpcError HasConflicts = new(-515, "Transaction has conflicts");
+        public static readonly RpcError UnknownError = new(-516, "Unknown error");
```

Comment:
Can't be, it's not a part of https://github.com/neo-project/proposals/blob/master/nep-23.mediawiki#verification-errors. If these errors don't suit existing codes, then `-500` should be used.

Reply:
I am overwrit this pr, please save ur time dont review yet